tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -344,10 +344,17 @@ def __str__(self):
 
 def targets():
     """Gets list of supported targets"""
-    return ([
-        Cls(platform, arch) for Cls in (CSharpExtArtifact, ProtocArtifact)
-        for platform in ('linux', 'macos', 'windows') for arch in ('x86', 'x64')
-    ] + [
+    return [
+        ProtocArtifact('linux', 'x64'),
+        ProtocArtifact('linux', 'x86'),
```

Comment:
Should the linux and macos x86 targets be removed? (I don't know what this file is about, so it may be fine - just checking.)

Reply:
Yes, that was the plan, see comment above.